.travis.yml

```diff
@@ -8,7 +8,6 @@ os:
 rust:
   # TODO: pick an oldest supported release
   - stable
-  - beta
```

Comment:
why remove beta?

Reply:
> Every six weeks, the latest nightly release is promoted to ‘Beta’. At that point, it will only receive patches to fix serious errors. Six weeks later, the beta is promoted to ‘Stable’, and becomes the next release of 1.x.

From https://doc.rust-lang.org/book/first-edition/release-channels.html#overview

Since we test from nightly to stable, I dont think beta is necessary.

Reply:
Got it.

Reply:
So if there is something wrong with beta, we need 6 weeks to get informed?

Reply:
I think patches are sent to nightly too. If those patches cause beta failure, nightly will also fail.

Reply:
From the same page, it seems that testing with three channels is recommanded:
>What about beta? We encourage all Rust users who use the stable release channel to also test against the beta channel in their continuous integration systems. This will help alert the team in case there’s an accidental regression.
>
>Additionally, testing against nightly can catch regressions even sooner, and so if you don’t mind a third build, we’d appreciate testing against all channels.